src/test/java/spoon/processing/CtGenerationTest.java

```diff
@@ -194,6 +196,53 @@ public void testGenerateRoleHandler() {
 		assertThat(actual).isEqualTo(expected);
 	}
 
+	@Test
+	@Disabled("only meant to be run manually to make copying easier")
+	void copyGeneratedFiles() throws IOException {
+		copy(
+				"./target/generated/spoon/support/visitor/replace/ReplacementVisitor.java",
+				"./src/main/java/spoon/support/visitor/replace/ReplacementVisitor.java"
+		);
+		copy(
+				"./target/generated/spoon/reflect/visitor/CtBiScannerDefault.java",
+				"./src/main/java/spoon/reflect/visitor/CtBiScannerDefault.java"
+		);
+		copy(
+				"./target/generated/spoon/support/visitor/clone/CloneBuilder.java",
+				"./src/main/java/spoon/support/visitor/clone/CloneBuilder.java"
+		);
+		copy(
+				"./target/generated/spoon/support/visitor/clone/CloneVisitor.java",
+				"./src/main/java/spoon/support/visitor/clone/CloneVisitor.java"
+		);
+		copy(
+				"./target/generated/spoon/reflect/meta/impl/ModelRoleHandlers.java",
+				"./src/main/java/spoon/reflect/meta/impl/ModelRoleHandlers.java"
+		);
+	}
+
+	private static void copy(String from, String to) throws IOException {
+		Files.copy(Path.of(from), Path.of(to), StandardCopyOption.REPLACE_EXISTING);
+	}
+
+	private void configurePrinter(Launcher launcher) {
+		Environment environment = launcher.getEnvironment();
+		environment.setPrettyPrinterCreator(new Supplier<>() {
+            @Override
+            public PrettyPrinter get() {
+				// we want to create a printer configured for the given environment,
+				// but we are who creates this printer - juggle around this StackOverflowError
+                environment.setPrettyPrinterCreator(null);
+                DefaultJavaPrettyPrinter printer = (DefaultJavaPrettyPrinter) environment.createPrettyPrinter();
+                environment.setPrettyPrinterCreator(this);
+                PrinterHelper printerHelper = new PrinterHelper(environment);
+                printerHelper.setPrefixBlockComments(true);
+                printer.setPrinterTokenWriter(new DefaultTokenWriter(printerHelper));
+                return printer;
+            }
+        });
+	}
+
```

Comment:
Can you use tabs here instead of spaces to be consistent and fix the indentation of the comment?